docs/source/examples/Drag and Drop.ipynb

```diff
@@ -0,0 +1,547 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Draggable Label"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "`DraggableLabel` is a label that can be dragged and dropped to other fields."
```

Comment:
It looks like `DraggableLabel` can be removed from the example notebook.

Reply:
Never mind that, I thought it was initially imported from `ipywidgets`

Reply:
Indeed, the initial implementation had a `DraggableLabel`, but we removed it in favor of the more general `DraggableBox`.